build.sbt

```diff
@@ -60,49 +27,12 @@ Global / onLoad := (Global / onLoad).value.andThen { s =>
   s
 }
 
-// this overrides interplay release settings with some adaptations for working with sbt-dynver
-// moreover, it contain only bits necessary for twirl, nothing else
-lazy val releaseSettings: Seq[Setting[_]] = Seq(
-  // Release settings
-  releaseCrossBuild := false,
-  releaseProcess := {
-    import ReleaseTransformations._
-
-    def ifDefinedAndTrue(key: SettingKey[Boolean], step: State => State): State => State = { state =>
-      Project.extract(state).getOpt(key in ThisBuild) match {
-        case Some(true) => step(state)
-        case _          => state
-      }
-    }
-
-    Seq[ReleaseStep](
-      checkSnapshotDependencies,
-      runClean,
-      releaseStepCommandAndRemaining("+test"),
-      releaseStepTask(playBuildExtraTests in thisProjectRef.value),
-      releaseStepCommandAndRemaining("+publishSigned"),
-      releaseStepTask(playBuildExtraPublish in thisProjectRef.value),
-      ifDefinedAndTrue(playBuildPromoteBintray, releaseStepTask(bintrayRelease in thisProjectRef.value)),
-      ifDefinedAndTrue(playBuildPromoteSonatype, releaseStepCommand("sonatypeBundleRelease")),
-      pushChanges
-    )
-  }
-)
-
-val commonSettings = javaCompilerSettings ++ headerSettings ++ Seq(
-  scalaVersion := scala212,
-  crossScalaVersions := Seq(scala212, scala213),
-  scalacOptions ++= scalacCompilerParams,
-)
-
 lazy val twirl = project
   .in(file("."))
-  .enablePlugins(PlayRootProject)
```

Comment:
I had to scratch my head on why `.enablePlugins(Common)` is not added here. But this looks good to me now.

Good job!

Reply:
I added Common, it's needed. I will push.

Reply:
why is it needed?

There is no artifact to produce, there's no docs or sources in `"."`. What am I missing?

Reply:
You are right, I added too much. 

`validateCode` is failing because this projection doesn't have headers. It previously include it, but I removed. I don't need to add the Common plugin just for it. I will solve in another way.

Reply:
@ignasi35, I did another push. 

We now have a CopyrightHeader plugin automatically added everywhere. When we call `validateCode`, that's a command alias and it will require that all projects (also aggregation projects) to have the header configure. If not, the validate fails. 

I took the path of other projects in Akka in which we add such a plugin and we don't try to disable things in aggregation projects. 

We still have another issue, the validation may is failing, but travis is ignoring it. I don't want to fix it right now. It was already so. I prefer to run manually and move this PR forward. Later I fix it. I just want to avoid a side track that can become a rabbit hole.